3p/messaging.js

```diff
@@ -25,7 +25,7 @@ export function nonSensitiveDataPostMessage(type, opt_object) {
   }
   const object = opt_object || {};
   object.type = type;
-  object.sentinel = window.context.amp3pSentinel;
+  object.sentinel = window.context.sentinel;
```

Comment:
ideally. less hassle for our release eng in case of issues.
a lesson we just learned: #6864